### PR TITLE
Fail tests on json error

### PR DESCRIPTION
**Pull request by martinratinaud**
It appears mocha does not throw `unhandledException` s and fails silently

This PR will fix this

Thanks @Ndpnt for the pair

### CODE REVIEW

File: src/archivist/services/index.js [hunk 1]
```diff
@@ -19,7 +19,14 @@ export async function load() {
...
 
   await Promise.all(serviceFileNames.map(async fileName => {
-    const serviceDeclaration = JSON.parse(await fs.readFile(path.join(declarationsPath, fileName)));
+    const jsonDeclarationFilePath = path.join(declarationsPath, fileName);
+    let serviceDeclaration;
+
+    try {
+      serviceDeclaration = JSON.parse(await fs.readFile(jsonDeclarationFilePath));
+    } catch (e) {
+      throw new Error(`JSON for document ${jsonDeclarationFilePath} could not be parsed`);
```
Comment (Ndpnt) [inline src/archivist/services/index.js:28]: I suggest:
`The "${path.basename(fileName, '.json')}" service declaration is malformed and cannot be parsed in ${jsonDeclarationFilePath}`

Example:
`The "Facebook" service declaration is malformed and cannot be parsed in /Users/ndpnt/Workspace/OTA/services-all/declarations/Facebook.json`

Comment (martinratinaud) [reply src/archivist/services/index.js]: I changed even though I believe reusing `path.basename(fileName, '.json')` is overkill

**Comment by martinratinaud** [comment]
Tests should pass once https://github.com/OpenTermsArchive/services-all/pull/524 is merged

**Comment by martinratinaud** [comment]
@Ndpnt I added your suggestions

**Review by Ndpnt — APPROVED** [review]

**Comment by martinratinaud** [comment]
Fixes #788

**Pull request merged**